nodejs/sqlcommenter-nodejs/packages/sqlcommenter-knex/provider/opentelemetry.js

```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const {context, defaultSetter} = require('@opentelemetry/api');
-const {HttpTraceContext} = require('@opentelemetry/core')
+const {context, defaultTextMapSetter} = require('@opentelemetry/api');
+const {HttpTraceContextPropagator} = require('@opentelemetry/core')
```

Comment:
Just a heads up, this was renamed to `W3CTraceContextPropagator` in an upcoming release